### PR TITLE
Proposed clarifications to the README file

### PR DESCRIPTION
**Pull request by goneall**
To resolve issue #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +22,13 @@ The following directories contain the license list in the format specified:
 
 The file licenses.md is a table of contents for the generated licenses.  The links for the files point to the text for the license.
 
+## Contributing
```
Comment (wking) [inline README.md]: I'd rather put this in a `CONTRIBUTING.md` to get [GitHub's increased visibility for that content][1].  We may also want [an issue template][2] with content like:

> Most of this repository is auto-generated.  Please see `CONTRIBUTING.md` for information on the appropriate upstream repositories.

[1]: https://help.github.com/articles/setting-guidelines-for-repository-contributors/#adding-a-contributing-file
[2]: https://help.github.com/articles/creating-an-issue-template-for-your-repository/

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # License List Data
 
+See [CONTRIBUTING.md](./CONTRIBUTING.md) for information on requesting new licenses, reporting issues or contributing pull requests related to this repository.
```
Comment (wking) [inline README.md]: You don't need the `./` here.  I'd prefer either:

```markdown
… [`CONTRIBUTING.md`][CONTRIBUTING.md] …
```

or:

```markdown
… [the contributing documentation][CONTRIBUTING.md] …
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Contributing to License List Data
+
+Technical questions or issues can be sent to the [SPDX technical team mailing list](mailto:spdx-tech@lists.spdx.org).
+Questions or issues on the license content can be sent to the [SPDX legal team mailing list](mailto:spdx-legal@lists.spdx.org).
+
+Note that the content of this repository is generated by the [SPDX tools](http://github.com/spdx/tools) from the [license list source](https://github.com/spdx/license-list-XML).
+
+As far as the *content* of the license information is concerned, see the [license-list-XML Contributing](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) document for information on how to report issues, request new licenses, or contribute changes.
```
Comment (wking) [inline CONTRIBUTING.md]: I think you should move “the” and “document” inside the link, so:

```markdown
…see [the license-list-XML contributing document](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) for …
```

And I'm ambivalent on “document” vs. “documentation”, but whichever you use, you should probably also use in the README if you use similar phrasing there.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+As far as the output *format* or the *generation* of the output files is concerned, any [issues](https://github.com/spdx/tools/issues) or [pull requests](https://github.com/spdx/tools/pulls) should be logged at the [SPDX tools](http://github.com/spdx/tools) project.
```
Comment (wking) [inline CONTRIBUTING.md]: Do these need to be updated to point at https://github.com/spdx/LicenseListPublisher?

Comment (goneall) [reply CONTRIBUTING.md]: Good catch!

I'll update

**Pull request merged**